pkg/reconciler/mtbroker/trigger.go

```diff
@@ -180,23 +179,11 @@ func (r *Reconciler) updateTriggerStatus(ctx context.Context, desired *v1alpha1.
 		return trigger, nil
 	}
 
-	becomesReady := desired.Status.IsReady() && !trigger.Status.IsReady()
-
 	// Don't modify the informers copy.
 	existing := trigger.DeepCopy()
 	existing.Status = desired.Status
 
-	trig, err := r.EventingClientSet.EventingV1alpha1().Triggers(desired.Namespace).UpdateStatus(existing)
-	if err == nil && becomesReady {
-		duration := time.Since(trig.ObjectMeta.CreationTimestamp.Time)
-		r.Logger.Infof("Trigger %q became ready after %v", trigger.Name, duration)
-		r.Recorder.Event(trigger, corev1.EventTypeNormal, triggerReadinessChanged, fmt.Sprintf("Trigger %q became ready", trigger.Name))
-		if err := r.StatsReporter.ReportReady("Trigger", trigger.Namespace, trigger.Name, duration); err != nil {
```

Comment:
Why get rid of this stat? Seems like a useful metric to see if things take a long time?

Reply:
they are not useful stats as the metric is 100% a lie. We dropped them from the genereconciler because if the resource toggles from ready to unready back to ready, it will be recorded as "took 42 days to go ready".

Reply:
makes sense :)